index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Williams David</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/style.css">
+
 </head>
+
 <body>
-    
+    <div class="image">
+        <nav>
+            <div class="navbar-center">
+                <ul>
+                    <li>
+                        <a href="/">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="/">About</a>
+                    </li>
+                    <li>
+                        <a href="/">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mobile">
+                <ul>
+                    <li><a>Welcome</a></li>
+                    <li>
+                        <a class="menu"><i class="fa-solid fa-bars"></i></a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+    <article class="heading">
+        <section class="title">
+            <h1>Hey there. I’m David</h1>
+               <h2 class="title2">I’m a software developer</h2>
+        </section>
+        <section class="subtitle">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>.
+        </section>
+        <div class="network">
+            <ul>
+                <li><a href="/"><i class="fa-brands fa-twitter-square"></i></a></li>
+                <li><a href="/"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="/"><i class="fa-brands fa-facebook-square"></i></a></li>
+                <li><a href="/"><i class="fa-brands fa-github-square"></i></a></li>
+            </ul>
+        </div>
+        <div class="scroll-down">
+            <a class="button" href="/"><i class="fa-solid fa-circle-chevron-down"></i></a>
+        </div>
+
+    </article>
```

Comment:
- [ ] Please wrap this section with `<section>` tad instead of `<article>`
- Explanations: `<section>` element can be used for Chapters, Introductions, News items, and Contact information.
- `<article>` element can be used for Forum posts, Blog posts, User comments, Product cards, and Newspaper articles.
[html5_semantic_elements.asp](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
okay perfect